### PR TITLE
Add SPA06 and SPL07  baro support for Kakute series FC

### PR DESCRIPTION
**Pull request by jamming**
The BARO used in the KAKUTE series flight controllers is BMP280. BMP280 is now discontinued. Need to replace the baro of KAKUTE series flight controllers with SPA06/SPL07.

### CODE REVIEW

**Comment by vincentpoont2** [comment]
@sugaarK We are hoping to add these default support for the Holybro Kakute series.

**Review by sugaarK — APPROVED** [review]

File: configs/KAKUTEF4/config.h [hunk 1]
```diff
@@ -34,6 +34,7 @@
 #define USE_ACC_SPI_ICM20689
 #define USE_BARO
 #define USE_BARO_BMP280
+#define USE_BARO_DPS310
```
Comment (ot0tot) [inline configs/KAKUTEF4/config.h:37]:
```suggestion

```
This FC has no I2C resources assigned, so these defines should not be included.

File: configs/KAKUTEF7/config.h [hunk 2]
```diff
@@ -33,6 +33,7 @@
 #define USE_GYRO_SPI_MPU6000
 #define USE_ACC_SPI_MPU6000
 #define USE_BARO_BMP280
+#define USE_BARO_DPS310
```
Comment (ot0tot) [inline configs/KAKUTEF7/config.h:36]:
```suggestion
#define USE_BARO
#define USE_BARO_BMP280
#define USE_BARO_DPS310
```

File: configs/KAKUTEF7HDV/config.h [hunk 3]
```diff
@@ -31,6 +31,7 @@
 #define USE_GYRO
 #define USE_GYRO_SPI_MPU6000
 #define USE_BARO_BMP280
+#define USE_BARO_DPS310
```
Comment (ot0tot) [inline configs/KAKUTEF7HDV/config.h:34]: Same comment as KAKUTEF7 target
```suggestion
#define USE_BARO
#define USE_BARO_BMP280
#define USE_BARO_DPS310
```

and remove
```
#define USE_BARO
#define DEFAULT_BARO_DEVICE BARO_BMP280
```

Comment (ot0tot) [reply configs/KAKUTEF7HDV/config.h:34]: Also remove `DEFAULT_BARO` defines in other targets.

**Comment by sugaarK** [comment]
> @sugaarK We are hoping to add these default support for the Holybro Kakute series.

no worries 
good you've moved to other units instead of the clone units available

**Comment by haslinghuis** [comment]
Continue in https://github.com/betaflight/config/pull/609

**Pull request closed without merging**